### ISSUE TITLE
documentation mentions artifactId without code

### ISSUE THREAD

**Issue by HennoVermeulen** [dropwizard-jobs/dropwizard-jobs#128]
The documentation mentions to use Maven `<artifactId>dropwizard-jobs</artifactId>`. I think this is a Maven multi-module project/parent pom with quartz and dropwizard dependencies but without any code. When depending on this, I don't get the class `JobsBundle` and the interface `JobConfiguration`. This is solved when I use 

`<artifactId>dropwizard-jobs-core</artifactId>`

I must say that I use Gradle to get the dependencies, I'm not sure if the same issue happens with Maven but I would not be surprised.

**Comment by pas-marcus** [comment]
Thank you @HennoVermeulen for sharing this work around. I have the same issue with my gradle project, and your solution has worked.

**Comment by jokyjang** [comment]
Same here. Thanks @HennoVermeulen

**Comment by firebiird** [comment]
Yes I agree. I think the documentation should be updated to be : 

``` 
<dependency>
    <groupId>io.github.dropwizard-jobs</groupId>
    <artifactId>dropwizard-jobs-core</artifactId>
    <version>4.0.0-RELEASE</version>
</dependency>
```

**Comment by hakandilek** [comment]
This is actually duplicate of #127 and intended to be fixed with #169.

Closing atm.
